### ISSUE TITLE
Update maxpower calculations to consider account-wide max power

### ISSUE THREAD

**Issue by bhollis** [DestinyItemManager/DIM#10489]
The algorithm for "drop power" has changed in The Final Shape, we should change our maxpower filters and the gear power menu to account for that.

**Comment by UndarkAido** [comment]
It's still useful to be able to see a character's max equipable/infusible power too, so it'd be nice to still be both

**Comment by bhollis** [comment]
Perhaps this should be a new search, `droppower:`?

**Comment by Prometheus3375** [comment]
I hit 2000 drop last week and can confirm that evaluation of the current maximum power level (which is later used to evaluate power level of new loot) uses both weapons and armor from the whole account rather than weapons account-wide and armor class-specific.

It would be nice to update the popup with gear power levels to reflect this value.
Here is a rough sketch.

![image](https://github.com/DestinyItemManager/DIM/assets/35541026/f3e10514-9359-4236-9124-5c0fa4e21f8a)

The first armor column is maximum equipable armor, the second is maximum armor overall. Probably these columns must be labeled.

**Comment by Prometheus3375** [comment]
And it looks like the issue with gear on other characters is still here. Essentially, while you play on char 1, but your max gear is on char 2, the game sometimes forgets this and uses only the gear on char 1, their postmaster and vault. Cheese Forever covered it [here](https://youtu.be/9MvVWd900AU?si=ovMC2z4Z2lIudvov). This sometimes is when you move max power gear (mostly weapons) via API to other chars.

I will use next notation as a shortcut.
[kinetic, energy, power, helmet, arms, chest, legs, class]
V - vault, h - hunter, w - warlock, t - titan

What just happened to me:
My gear: [1998 V, 1999 V, 2000 V, 1998 V, 1999 w, 1999 w, 1999 w, 2000 w]
I did Warlords Ruin on titan and all my encounter loot was 2000 as expected. I got legs, helmet and kinetic.
My gear: [**2000 t**, 1999 V, 2000 V, **2000 t**, 1999 w, 1999 w, **2000 t**, 2000 w]
Then I moved 1998 kinetic from vault to titan and dismantled it. I accidentally moved my 1999 energy and also dismantled it.
My gear: [2000 t, **1989** V, 2000 V, 2000 t, 1999 w, 1999 w, 2000 t, 2000 w]
The new average is 1998 3/8. Given this I was not very upset with my mistake, and I knew how to fix it. I usually hoard items in previous season passes to align power level in a new season as the loot from season passes always drops at current maximum power level. I pulled Ex Diris from Witch's pass, but forgot to change recipient from my hunter to titan. Ex Diris was 1997, not 1998.

This happened because I moved my 1998 kinetic and 1999 energy from vault to titan via API (ex. DIM) and then dismantled them. Game registered neither my new 2000 kinetic and probably nor any other new 2000 loot for hunter. In that case my max gear for hunter was [**1995 h**, 1989 V, 2000 V, **1998 V**, 1999 w, 1999 w, **1999 w**, 2000 w] with average 1997 3/8 which matches Ex Diris PL.
